### ISSUE TITLE
Support for 4.6 / 4.7 ?

### ISSUE THREAD

**Issue by rkerr** [PowerDNS/pdns_recursor-ansible#76]
Looks like 4.5 is the latest version this ansible role knows about? could do with updated vars for 4.6 and 4.7?

**Comment by Algirdas-Z** [comment]
```
pdns_rec_install_repo:
  name: "pdns-recursor-47" # the repository name
  gpg_key: "https://repo.powerdns.com/FD380FBB-pub.asc" # repository public GPG key
  gpg_key_id: "FD380FBB-pub" # to avoid to reimport the key each time the role is executed
  yum_repo_baseurl: "https://repo.powerdns.com/centos/$basearch/$releasever/rec-47"```

Just add needed repo and go for it :)

**Comment by npmdnl** [comment]
Closed by #77
